### PR TITLE
[9.0][MIG] Backport of project_task_add_very_high

### PR DESCRIPTION
**Pull request by astirpe**
Notice that this module contains a migration script from V8 to V9, since the `priority` field was changed on standard project module by Odoo

### CODE REVIEW

File: project_task_add_very_high/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Onestein (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Project Task Add Very High",
+    "summary": "Adds an extra option 'Very High' on tasks",
+    "version": "9.0.1.0.0",
+    "author": "Onestein",
```
Comment (dreispt) [inline project_task_add_very_high/__openerp__.py]: add the OCA

File: project_task_add_very_high/migrations/9.0.1.0.0/post-migration.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Onestein (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+def migrate(cr, version):
+    # convert priority to avoid inconsistency
+    cr.execute(
+        "UPDATE project_task SET priority = '2' WHERE priority like '3'"
```
Comment (dreispt) [inline project_task_add_very_high/migrations/9.0.1.0.0/post-migration.py:9]: :trophy:

**Review by dreispt — COMMENTED** [review]
One small change and it's good to go.

**Review by dreispt — APPROVED** [review]
To squash merge.

**Comment by espo-tony** [comment]
:+1:

**Pull request merged**